src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,28 @@
+package duke.task;
+
+/**
+ * Representation of most basic task with just a task description
+ */
+public class Todo extends Task {
+    public Todo(String task) {
+        super(task);
+    }
+
+    public Todo(boolean done, String task) {
+        super(task);
+        this.done = done;
+    }
+
+    /**
+     * Returns a String representation of this Todo for storage to a file.
+     * @return String representation
+     */
+    public String fileString() {
```

Comment:
Encouraged to use verbs for method names! Maybe example getFileString()?

Reply:
Ah good one, maybe toFileString() to be inline with toString()